woodwork/tests/accessor/test_table_accessor.py

```diff
@@ -162,6 +162,21 @@ def test_init_accessor_with_schema_errors(sample_df):
         iloc_df.ww.init(schema=schema)
 
 
+def test_accessor_with_schema_parameter_warning(sample_df):
+    xfail_dask_and_koalas(sample_df)
+
+    schema_df = sample_df.copy()
+    schema_df.ww.init(name='test_schema', semantic_tags={'id': 'test_tag'}, index='id')
+    schema = schema_df.ww.schema
+
+    head_df = schema_df.head(2)
+    assert head_df.ww.schema is None
+
+    warning = "A schema was provided and the following parameters were ignored: index, semantic_tags"
+    with pytest.warns(Warning, match=warning):
+        head_df.ww.init(index='id', semantic_tags={'id': 'test_tag'}, schema=schema)
+
```

Comment:
Let's also check that the head_df name and semantic tags are what we expect, since the initialization should still succeed even though we warn the user.
```
    assert head_df.ww.name == 'test_schema'
    assert head_df.ww.semantic_tags['id'] == {'index', 'test_tag'}
```